src/components/presentation/ArticleCard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const ArticleCard = ({ article }) => {
+  const defaultImg = 'https://tmsnrt.rs/2FNDBT2';
```

Comment:
I wouldn't want you exposing this. What about adding it to the .env file and calling it here as maybe process.env.REACT_DEFAULT_IMG?